tests/deployments/ansible/images/Dockerfile.centos6

```diff
@@ -0,0 +1,17 @@
+FROM centos:6
```

Comment:
Drop centos6 and Amazon Linux 1 since we are already planning on dropping support for them.  No use wasting CI run time on them.